cfgov/templates/wagtailadmin/css/table-block.css

```diff
@@ -1,3 +1,23 @@
+.Draftail-Editor__wrapper {
+ height: auto !important;
+ width: 800px !important;
+ max-width: 800px !important;
+}
+
+.Draftail-Editor .DraftEditor-root {
+    padding: 0.75em;
+}
+
+.handsontableInputHolder {
+  left: 0 !important;
+}
```

Comment:
```suggestion
.handsontableInputHolder {
  left: 0 !important;
  margin-top: -67px;
}
```

It's a little magic number-y, but adding a negative top margin here brings the editor up so that the editor text is right over the cell text and your cursor isn't right on top of a Draftail toolbar button when the editor appears. It's a minor thing but kind of nice to not have to move the cursor too much to start typing or have an inadvertent button click. I don' feel strongly about this if we'd rather not add the complexity, though.